src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -337,9 +329,16 @@ def handle_update_incident_project_select_action(
 
 # COMMANDS
 def handle_list_incidents_command(
-    body: dict, payload: dict, db_session: Session, context: BoltContext, client: WebClient
+    ack: Ack,
+    body: dict,
+    payload: dict,
+    db_session: Session,
+    context: BoltContext,
+    client: WebClient,
 ) -> None:
     """Handles the list incidents command."""
+    ack()
```

Comment:
Are these acks necessary if we've already returned a 200 ok? Are they simply a way to avoid bolt warning us that we haven't called `ack`?

Reply:
The latter, they're necessary to avoid bolt warning that we "haven't called ack()" and then exiting: https://github.com/slackapi/bolt-python/blob/c99c23fd056d26f8b1e39436bd1fcd2c83a3e1bd/slack_bolt/listener/thread_runner.py#L162-L168

We could still use the ack() middleware for this. I personally prefer this pattern but neither here nor there 🤷‍♀️